app/templates/events/event_list.html

```diff
@@ -106,10 +110,79 @@ <h2 class="accordion-header" id="accordion__header_{{program}}">
           </div>
         </div>
       </div>
+      {{emailModal}}
     {% endfor %}
   </div>
 {% endmacro %}
 
+{% set emailModal %}
```

Comment:
separate this modal into its own file and include it. This way others can also include the email modal.